presto-main/src/main/java/io/prestosql/sql/planner/plan/IndexJoinNode.java

```diff
@@ -65,7 +65,7 @@ public IndexJoinNode(
 
         private final String joinLabel;
 
-        private Type(String joinLabel)
```

Comment:
This is indeed redundant, but it seems we do this by convention pretty much for most of enums, i guess

Reply:
I would drop this. And drop others as we go. Note that in this PR I only looked at `presto-main`. There is a chance I will go over the others from time to time as a relaxation execrcise.